src/tox_travis/hooks.py

```diff
@@ -17,5 +20,16 @@ def tox_addoption(parser):
 @tox.hookimpl
 def tox_configure(config):
     """Check for the presence of the added options."""
+    if 'TRAVIS' not in os.environ:
+        return
+
     if config.option.travis_after:
         travis_after_monkeypatch()
+
+    # When `obey_outcomes` is set to `False`, this will pass the return code from tox to travis
+    # regardless of how `ignore_outcome` was configured.
+    tox_config = py.iniconfig.IniConfig('tox.ini')
+    travis_reader = tox.config.SectionReader("travis", tox_config)
+    if travis_reader.getbool('unignore_outcomes', False):
+        for env in config.envlist:
+            config.envconfigs[env].ignore_outcome = False
```

Comment:
Put this into a function in `toxenv.py`, and just call the function from here. This file should be the minimal shims required to interface with Tox hooks.